src/core/helpers/http-status.ts

```diff
@@ -1,3 +1,26 @@
+export const OK = 200;
+export const CREATED = 201;
+export const ACCEPTED = 202;
+export const NO_CONTENT = 204;
+export const MOVED_PERMANENETLY = 301;
+export const SEE_OTHER = 303;
+export const NOT_MODIFIED = 304;
+export const TEMPORARY_REDIRECT = 307;
+export const INVALID_JSON = 400;
+export const AUTHORIZATION_FAILURE = 401;
+export const BAD_REQUEST = 401;
+export const UNAUTHORIZED = 402;
+export const FORBIDDEN = 403;
+export const NOT_FOUND = 404;
+export const METHOD_NOT_ALLOWED = 405;
+export const NOT_ACCEPTABLE = 406;
 export const DUPLICATE_RESOURCE = 409;
+export const PRECONDITION_FAILED = 412;
+export const UNSUPPORTED_MEDIA_TYPE = 415;
+export const IM_A_LITTLE_TEAPOT = 418;
 export const INVALID_BODY_PARAMETER = 422;
+export const SSL_CERTIFICATE_ERROR = 495;
```

Comment:
This is not standard (it's idiomatic to Nginx) -- Nginx is non-essential to Sakura, so let's leave this out.